tools/android.py

```diff
@@ -11,20 +11,32 @@ def options(opts):
         "18" if "32" in ARGUMENTS.get("arch", "arm64") else "21",
     )
     opts.Add(
-        "ANDROID_NDK_ROOT",
-        "Path to your Android NDK installation. By default, uses ANDROID_NDK_ROOT from your defined environment variables.",
-        os.environ.get("ANDROID_NDK_ROOT", None),
+        "ANDROID_HOME",
+        "Path to your Android SDK installation. By default, uses ANDROID_HOME from your defined environment variables.",
+        os.environ.get("ANDROID_HOME", os.environ.get("ANDROID_SDK_ROOT")),
```

Comment:
I noticed on the Godot PR this changed to:

```suggestion
        os.environ.get("ANDROID_HOME", os.environ.get("ANDROID_SDK_ROOT", "")),
```

Reply:
I chose not to use an empty string as fallback because in core we explicitly check that the given path exists so an empty string would be invalid, whereas in the `android.py` logic, we check against `None`:
- https://github.com/godotengine/godot-cpp/blob/86dbd5fa0d6f581666e0626fe99bc2f2b00b5e49/tools/android.py#L21
- https://github.com/godotengine/godot-cpp/blob/86dbd5fa0d6f581666e0626fe99bc2f2b00b5e49/tools/android.py#L30
- https://github.com/godotengine/godot-cpp/blob/86dbd5fa0d6f581666e0626fe99bc2f2b00b5e49/tools/android.py#L37

Reply:
Ah, ok